pkg/features/kube_features.go

```diff
@@ -258,8 +258,9 @@ var defaultVersionedFeatureGates = map[featuregate.Feature]featuregate.Versioned
 	TASProfileMixed: {
 		{Version: version.MustParse("0.11"), Default: false, PreRelease: featuregate.Deprecated},
 	},
-	HierarchialCohorts: {
-		{Version: version.MustParse("0.12"), Default: true, PreRelease: featuregate.Beta},
+	HierarchicalCohorts: {
+		{Version: version.MustParse("0.7"), Default: false, PreRelease: featuregate.Alpha},
```

Comment:
This isn't true though.

We introduced the feature gate in 0.12..

Reply:
Sorry, I missed that, could you please correct it @tenzen-y . Actually we can claim it was introduced in 0.11 because it was cherry picked

Reply:
That makes sense. We can handle the feature gate version as a separate from actual feature versions: https://github.com/kubernetes-sigs/kueue/blob/e9b35497ccf5b0534cce64a2a5f71c81b0926d6d/keps/79-hierarchical-cohorts/kep.yaml#L21-L24